src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.vb

```diff
@@ -348,6 +348,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                     If tokenKind = SyntaxKind.None OrElse format.CompilerInternalOptions.IncludesOption(SymbolDisplayCompilerInternalOptions.UseMetadataMethodNames) Then
                         builder.Add(CreatePart(SymbolDisplayPartKind.MethodName, symbol, symbol.Name, visitedParents))
+                    ElseIf SyntaxFacts.IsKeywordKind(tokenKind) Then
```

Comment:
>ElseIf SyntaxFacts.IsKeywordKind(tokenKind) Then [](http://example.com/codeflow?start=20&length=48)

Is this ```ElseIf``` necessary?

Reply:
If I remove it, then keywords like 'And' get classified as operators. Vb ide has always classified the words as keywords and only the sigils as operators, so this preserves that behavior.

Reply:
sure.